molgenis-metadata-manager/src/main/frontend/src/components/generic-components/AttributeTreeNode.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <!-- TODO: Buttons for changing order -->
+  <li>
+    <span v-bind:class="{ 'selected-attribute-node': attribute.selected }" @click="onAttributeSelect(attribute)">
+      <i v-bind:class="['fa', isFolder ? 'fa-folder-o' : 'fa-columns']"></i> {{attribute.label}}
+    </span>
+    <ul v-if="isFolder">
+      <attribute-tree-node v-for="child in attribute.children" :attribute="child" :onAttributeSelect="onAttributeSelect"></attribute-tree-node>
+    </ul>
+  </li>
+</template>
+
+<style scoped>
+  li {
+    list-style-type: none;
+  }
+
+  .selected-attribute-node {
+    background-color: #c4e3f3;
```

Comment:
this seems like a 'magic number' you could change this to use a sass variable and select a value from the color scheme we are using. This makes is possible to change the theme op the app.